checkstyle.xml

```diff
@@ -65,10 +65,6 @@
       <property name="max" value="100"/>
     </module>
     <module name="MethodLength"/>
-    <module name="ParameterNumber">
```

Comment:
Removed as per @JakeWharton comment earlier.  It now failed on the CommandLineOptions constructor.